inactivity.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+title: Maintainer and Repository Inactivity
+nav_order: 6
+---
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+# Inactivity Policies
+
+As part of the normal lifecycle of a project codebases and maintainers come and go.  This document formalizes a Hyperledger wide polciy for archiving projects and moving maintainers to emereitus status when a proejct does not have an explicit policy of their own.
+
+## Maintainer Inactivity
+
+Hyperledger very much appreciates the contributions of all maintainers but removing write privledges is in the interest of an orderly and secure project.
+
+Maintainers who have not contributed to a Hyperledger project for (`exact timeframe to be decided`) 3 / 4 / 6 months will be removed from the relevant Github Organizations.
```

Comment:
A few questions that may arise, just thinking out loud here
1. Repository never needed a maintainer activity.
2. The maintainer is responsible for multiple projects, the maintainer did not have a commit or an activity in one of those projects in the last [3/4/6] months.
3. Should the access be removed from the organization or only from the repository that the user is a maintainer of. If a maintainer is not part of any project then eventually be removed from the org.

Reply:
1. ?? I'm not sure what is being asked ??
2. The tooling appears to allow us to track per-repo, so we can enforce this on a per-project basis. i.e. if I am part of project Foo and project Bar and stop contributing to Bar for 6 months, my maintainer status on Bar is removed and my commit access for Bar is removed.  My status in Foo is unaffected
3. as discussed further down the list, it will be per team. If they are removed from all teams then org removal would be the next step.

Reply:
For item (1) I was referring to a highly possible case, let's consider a project, where one of the maintainer is responsible only to maintain a specific repository. But it happened so that the repository is pretty much stable and did not require any update for months.